v2/index.html

```diff
@@ -1358,14 +1358,40 @@ <h4>Chords</h4>
             <div class="issue" data-number="60"></div>
             <div class="issue" data-number="65"></div>
             <p>
-                Enter chords from the highest to the lowest note, each one separated by <code>^</code>.
+                Notes in a chords are enclosed in the characters <code>^</code> and <code>></code>.
+            </p>
+            <p>
+               A <a>duration</a> value MAY precede the <code>^</code>. Two or more <a>note names</a>
+               (with optional octave and accidental indications) MUST immediately follow this character.
+               The notes MUST be ordered from the lowest to the highest on the staff line. A <code>></code>
+               MUST be used to end the chord. A duration value for the individual notes within a chord MUST
+               NOT be supplied.
+            </p>
+            <p>
+                If the duration is omitted, the last specified duration is used. If no duration is
+                supplied in the <a>encoding</a> a default duration of <code>4</code> is assumed.
+            </p>
+            <p>
+                A <code>p</code> character MAY follow the duration to indicate the chord has a fermata.
+            </p>
+            <p>
+                A <code>v</code> MAY be used to indicate a tied chord. A duration value MAY immediately
+                precede the <code>v</code>. If the duration is omitted, the last specified duration is
+                used. If no duration is supplied in the <a>encoding</a> a default duration of <code>4</code>
+                is assumed.
```

Comment:
Would this also require the `_`? That is `4^CEG>_v`? I would expect it because it makes it clearer.

Reply:
No, the `_` is not necessary in this scheme. I was toying with the idea of using it, but it seemed like it might cause problems because it wouldn't function the same as ties, as they are currently specified, do. 

I want, effectively, a "repeat the chord" character, so that there is no possibility that the chords on either end of the tie are different. That's what the `v` gives. 

For notes, the current method means you have to specify the note on the other end. `F_F`. If we wanted to apply this to the chord, then I would expect `^CEG>_^CEG>`, not just `^CEG>_v`. Also, the `v` might get confused as a repeat character for the chord... it's not, strictly speaking, since you can't simply do ^CEG>v` and get a repeated chord with no ties.

Reply:
A new proposal is in #112 that would unify the behaviour of ties, and allow the use of `_` on chords.